agent/tools/symlinks_test.go

```diff
@@ -57,15 +57,19 @@ func (s *SymlinksSuite) TestEnsureSymlinksSymlinkedDir(c *gc.C) {
 }
 
 func (s *SymlinksSuite) testEnsureSymlinks(c *gc.C, dir string) {
+	// If we have both 'jujuc' and 'jujud' prefer 'jujuc'
```

Comment:
Is there a test with just jujud where we check the links?

Reply:
not currently, we probably should add it.